### DISCUSSION TITLE
我阿凡达下了两部，硬链接也对，这里只有一部，怎么回事

### DISCUSSION THREAD

**Discussion by xixi889988** [jxxghp/MoviePilot#2028] [问题咨询]
我阿凡达下了两部，硬链接也对，这里只有一部，怎么回事
<img width="699" alt="c968db5d1d29e7a4a13952f8fe2bd352" src="https://github.com/jxxghp/MoviePilot/assets/169155893/8489ad11-73b9-41b9-a2be-226c2ee3f236">
<img width="1024" alt="750b371720b2d01ff8bce07063ef251c" src="https://github.com/jxxghp/MoviePilot/assets/169155893/3eb0a501-8cf6-4550-a236-1769676f0756">
<img width="810" alt="5d1e1c8261d851777f086f7d6ea7989a" src="https://github.com/jxxghp/MoviePilot/assets/169155893/c06b689b-0697-46fb-9cc5-803e31471618">

**Comment by cng10001** [comment]
因为你硬链接的重命名规则加的特征项太少了，没法区分原文件名上的信息
